server/gendb/bindata.go

```diff
@@ -109,7 +109,7 @@ func dynamodbGoTmpl() (*asset, error) {
 	}
 
 	info := bindataFileInfo{name: "dynamodb.go.tmpl", size: 0, mode: os.FileMode(0), modTime: time.Unix(0, 0)}
-	a := &asset{bytes: bytes, info: info, digest: [32]uint8{0xc0, 0xa3, 0xce, 0x3d, 0xab, 0xfc, 0x2, 0xea, 0x20, 0x3, 0x0, 0x1e, 0xb, 0x26, 0x27, 0x54, 0x67, 0xb5, 0xef, 0x2f, 0x52, 0x94, 0x9d, 0x97, 0xed, 0x90, 0x3, 0x9d, 0xa4, 0x52, 0xe3, 0x50}}
+	a := &asset{bytes: bytes, info: info, digest: [32]uint8{0x5a, 0x84, 0x42, 0xe8, 0x33, 0xfe, 0x9, 0x3, 0x22, 0xfb, 0x16, 0xa3, 0x5a, 0xad, 0xd6, 0x3d, 0x98, 0xa, 0x70, 0xb3, 0x41, 0x4b, 0xf8, 0x39, 0xf3, 0x7, 0xc6, 0xba, 0xd4, 0x2e, 0x2a, 0xcf}}
```

Comment:
Question: What is this change doing? Thanks in advance!

Reply:
I believe it is the binary embedding for the template. You can read more about go-bindata in [this repo](https://github.com/go-bindata/go-bindata)

Reply:
Thank you!! @dxu429